src/ofxstatement/parser.py

```diff
@@ -43,18 +43,29 @@ def parse(self) -> Statement:
             self.cur_record += 1
             if not line:
                 continue
-            stmt_line = self.parse_record(line)
-            if stmt_line:
-                stmt_line.assert_valid()
-                self.statement.lines.append(stmt_line)
+            # OFX importer plugins don't support 100% of OFX spec, so in certain cases
+            # (ex: transfer + bank fees, or income + federal tax), it's easier to export
+            # a single line from statement as 2 or more transactions
+            parsed_stmt_lines = self.parse_record(line)
+            if parsed_stmt_lines:
```

Comment:
Hm, I'm not a big fan of functions that return different types. I think we shouldn't do change the interface here - if a particular file format doesn't fit into "one line per record" pattern, the plugin can simply override the `parse` method and do whatever needs to be done to create a `Statement` object (IOW, extend `AbstractStatementParser` instead of `StatementParser`).